.github/workflows/build_and_test.yml

```diff
@@ -56,7 +56,10 @@ jobs:
         if: runner.os == 'Linux'
         run: ./scripts/check_for_blobs.sh
       - name: Build libafl debug
+        if: runner.os == 'Linux'
         run: cargo build -p libafl
+      - name: Build libafl book
```

Comment:
better separate this to another job

Reply:
instead of adding to this exsiting one. i mean. then you run things in parallel

Reply:
this takes no time anyway, not worth the effort

And anyway we need to move all CI to justfile asap

Reply:
ok